src/ilasm/writer.cpp

```diff
@@ -982,6 +982,59 @@ BOOL Assembler::EmitEventsProps(Class* pClass)
     return ret;
 }
 
+HRESULT Assembler::AllocateStrongNameSignature()
+{
+    HRESULT             hr = S_OK;
+    HCEESECTION         hSection;
+    DWORD               dwDataLength;
+    DWORD               dwDataOffset;
+    DWORD               dwDataRVA;
+    VOID               *pvBuffer;
+    AsmManStrongName   *pSN = &m_pManifest->m_sStrongName;
+
+    int keySize = sizeof(pSN->m_pbPublicKey)/sizeof(BYTE);
```

Comment:
This does not look right: `m_cbPublicKey` is pointer. `sizeof(pSN->m_cbPublicKey)` will be size of pointer...